test/CadlRanchProjects/special-words/tspconfig.yaml

```diff
@@ -0,0 +1,4 @@
+options:
+  "@azure-tools/typespec-csharp":
+    use-overloads-between-protocol-and-convenience: false
+    model-namespace: false
```

Comment:
Exclude the model namespace because there is a client named as `Models` which will cause conflict when the model namespace is applied.
Current we suppress this by excluding the model-namespace, and filed an issue https://github.com/Azure/autorest.csharp/issues/3847 to resolve in future.